src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs

```diff
@@ -76,11 +76,14 @@ public override async Task OnConnectedAsync(HubConnectionContext connection)
 
         var hubActivator = scope.ServiceProvider.GetRequiredService<IHubActivator<THub>>();
         var hub = hubActivator.Create();
+        Activity? activity = null;
         try
         {
             // OnConnectedAsync won't work with client results (ISingleClientProxy.InvokeAsync)
             InitializeHub(hub, connection, invokeAllowed: false);
 
+            activity = CreateActivity(scope.ServiceProvider, nameof(hub.OnConnectedAsync));
```

Comment:
Are the `OnConnectedAsync` and disconnected invocations always significant? Would it make sense to only report them if the methods are overridden?

Reply:
They're significant in the sense that they describe when the connection is usable. So for example if the handshake took a long time you could see the gap between the connection starting and `OnConnectedAsync` starting where it's not usable yet.

Reply:
I think we should keep them